playbooks/roles/kibana/tasks/kibana_post_install.yml

```diff
@@ -0,0 +1,135 @@
+---
+# Copyright 2014, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+- name: create self-signed SSL cert
+  command: >
+    openssl req -new -nodes -x509 -subj
+    "/C=US/ST=Texas/L=San Antonio/O=IT/CN={{ kibana_server_name }}"
+    -days 365
+    -keyout /etc/ssl/private/apache.key
+    -out /etc/ssl/certs/apache.cert
+    -extensions v3_ca
+    creates=/etc/ssl/certs/apache.cert
+  when: kibana_self_signed is defined and kibana_self_signed == true
+  tags:
+    - kibana-self-signed-cert
+    - kibana-post-install
+
+- name: Enable apache modules
+  command: a2enmod "{{ item }}"
+  with_items: kibana_apache_modules
+  tags:
+    - kibana-apache-modules
+    - kibana-apache
+    - kibana-post-install
+
+- name: Template Kibana Apache Config
+  template:
+    src: "{{ item }}"
+    dest: "/etc/apache2/sites-available/{{ item }}"
+    owner: "{{ kibana_system_user }}"
+    group: "{{ kibana_system_group }}"
+  with_items:
+    - 000-kibana.conf
+  notify: Restart Apache
+  tags:
+    - kibana-apache
+    - kibana-post-install
+
+- name: Fix Apache2 Paths
```

Comment:
I've compared the current trusty apache2.conf with the template we're carrying and they seem to be exactly the same. I recommend that we remove this file from the role.